.github/tinytex.yml

```diff
@@ -0,0 +1,54 @@
+# This is a basic workflow that is manually triggered
+
+ name: Archive TinyTex
+
+ # Controls when the action will run. Workflow runs when manually triggered using the UI
+ # or API.
+ on:
+   workflow_dispatch:
+     # Inputs the workflow accepts.
+     inputs:
+       version:
+         # Friendly description to be shown in the UI instead of 'name'
+         description: 'TinyTex Version to Archive'
+         # Default value if no value is explicitly provided
+         default: ''
+         # Input has to be provided for the workflow to run
+         required: true
+
+ # A workflow run is made up of one or more jobs that can run sequentially or in parallel
+ jobs:
+   # This workflow contains a single job called "greet"
+   archive-tinytex:
+     env:
+      EXTS: tar.gz tgz zip
+      TARGET_DIRECTORY: tinytex/${{ github.event.inputs.version }}/
+     # The type of runner that the job will run on
+     runs-on: ubuntu-latest
+
+     # Steps represent a sequence of tasks that will be executed as part of the job
+     steps:
+     # Runs a single command using the runners shell
+     - name: Download Local Copies
+       run: |
+            for EXT in $EXTS; do
+             FILENAME=TinyTeX-${{ github.event.inputs.version }}.$EXT
+             curl -fail -L -o $FILENAME https://github.com/yihui/tinytex-releases/releases/download/${{ github.event.inputs.version }}/$FILENAME
```

Comment:
Which variation are we targeting? (looks like there are 3: https://github.com/yihui/tinytex-releases#releases)

Reply:
I targeted the `TinyTeX.*` with this, but looking merely closely, I think we could get away with `TinyTeX-1.*` which would be enough to compile most RMarkdown documents. What do you think?